docs/GettingStarted.md

```diff
@@ -448,6 +450,63 @@ You can also get the current active trace using the `active_trace` method. This
 current_trace = Datadog::Tracing.active_trace
 ```
 
+## Method Tracing API
+Another option for manual instrumentation is to use the method tracing API to trace your own code or code from a library/framework we do not yet support.
+
+To generate traces, use the `Datadog::Tracing.trace_method` method:
+```ruby
+Datadog::Tracing.trace_method(name, target, span_options).around do |env, span, trace, &block|
```

Comment:
Does this need to be updated to reflect `target, name` instead of `name, target`?